src/MLJMultivariateStatsInterface.jl

```diff
@@ -150,4 +147,1102 @@ metadata_pkg.(
+ `in_dim` and `out_dim` are the input and output dimensions respectively.
+- `priors`: The class priors for classification. As inferred from training target `y`,
+   if not user-specified. A vector with order consistent with `levels(y)`.
+
+# Report
+
+The fields of `report(mach)` are:
+
+- `classes`: The classes seen during model fitting.
+- `out_dim`: The dimensions the model is projected to.
+- `class_means`: The matrix comprised of class-specific means as
+  columns (see above).
+- `mean`: The mean of the untransformed training data, of length `in_dim`.
+- `class_weights`: The weights of each class.
+- `Sb`: The between class scatter matrix.
+- `Sw`: The within class scatter matrix.
+- `nc`: The number of classes directly observed in the training data (which can be
+   less than the total number of classes in the class pool)
+
+# Examples
+
+```
+using MLJ
+
+BLDA = @load BayesianLDA pkg=MultivariateStats
+
+X, y = @load_iris
+
+model = BLDA()
+mach = machine(model, X, y) |> fit!
+
+Xproj = transform(mach, X)
+y_hat = predict(mach, X)
+labels = predict_mode(mach, X)
+```
+
+See also
+[`LDA`](@ref), [`SubspaceLDA`](@ref), [`BayesianSubspaceLDA`](@ref)
+"""
+BayesianLDA
+
+"""
+
+$(MMI.doc_header(SubspaceLDA))
+
+`SubspaceLDA`: Multiclass subspace linear discriminant analysis (LDA) is a variation on
+ordinary LDA suitable for high dimensional data, as it avoids storing scatter
+matrices. For details, refer the
+[MultivariateStats.jl documentation](https://juliastats.org/MultivariateStats.jl/stable/).
+In addition to dimension reduction, probabilistic classification is provided.
+In the case of classification, the class probability for a new observation
+reflects the proximity of that observation to training observations
+associated with that class, and how far away the observation is from those
+associated with other classes. Specifically, the distances, in the transformed
+(projected) space, of a new observation, from the centroid of each target class,
+is computed; the resulting vector of distances (times minus one) is passed to a
+softmax function to obtain a class probability prediction. Here "distance"
+is computed using a user-specified distance function.
+
+# Training data
+
+In MLJ or MLJBase, bind an instance `model` to data with
+
+    mach = machine(model, X)
+
+Where
+
+- `X`: is any table of input features (eg, a `DataFrame`) whose columns
+  are of scitype `Continuous`; check the scitypes with `schema(X)`
+- `y`: is the target, which can be any `AbstractVector` whose element
+  scitype is `OrderedFactor` or `Multiclass`; check the scitype
+  with `scitype(y)`
+
+Train the machine using `fit!(mach, rows=...)`.
+
+# Hyper-parameters
+
+- `normalize=true`: Option to normalize the between class variance for the number of
+   observations in each class, one of `true` or `false`.
+- `out_dim`: The dimension of the transformed space to be used by `predict` and
+   `transform` methods, automatically set if 0 is given (default).
```

Comment:
```suggestion
- `outdim=0`: The dimension of the learned lower dimensional feature space. Automatically set if 0.
```

Reply:
@ablaom we need to tell the users that `out_dim` is used by `predict/transform` methods.